pkg/ledengine/ledengine_test.go

```diff
@@ -113,33 +113,34 @@ func TestNewBurstPattern(t *testing.T) {
 	}
 }
 
-func TestNewSlowBlinkPattern(t *testing.T) {
-	type args struct {
-		baseColor   led.Color
-		activeColor led.Color
-	}
-	tests := []struct {
-		name string
-		args args
-		want ledengine.BlinkPattern
-	}{
-		// TODO: Add test cases.
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			if got := ledengine.NewSlowBlinkPattern(tt.args.baseColor, tt.args.activeColor); !reflect.DeepEqual(
-				got,
-				tt.want,
-			) {
-				t.Errorf("NewSlowledengine.BlinkPattern() = %v, want %v", got, tt.want)
-			}
-		})
-	}
-}
+// FIXME: No dead code
+//func TestNewSlowBlinkPattern(t *testing.T) {
```

Comment:
Let's just remove the code for this; I've been contemplating to use this instead of the current blinking pattern because RC2 revision blades had sometime quirks with the timing of the WS2812 protocol (the V1 revision does not show the same behavior, could be fixed by a rerouting of the connections, different thickness of ground-plane, etc)